lib/auth/machineid/workloadidentityv1/issuer_service.go

```diff
@@ -470,16 +471,24 @@ func baseEvent(
 }
 
 func calculateTTL(
+	ctx context.Context,
+	log *slog.Logger,
 	clock clockwork.Clock,
 	requestedTTL time.Duration,
+	configuredMaxTTL time.Duration,
 ) (time.Time, time.Time, time.Time, time.Duration) {
-	ttl := time.Hour
-	if requestedTTL != 0 {
-		ttl = requestedTTL
-		if ttl > defaultMaxTTL {
-			ttl = defaultMaxTTL
-		}
+	ttl := cmp.Or(requestedTTL, time.Hour)
```

Comment:
💅 
We could extract the default value (time.Hour) and have it right next top `defaultMaxTTL` , with a comment explaining that the `defaultTTL` should never be greater than the `defaultMaxTTL`

Reply:
9a857ca9ea